lib/web_ui/lib/ui.dart

```diff
@@ -49,7 +49,9 @@ void webOnlyInitializeEngine() {
   engine.initializeEngine();
 }
 
-void webOnlySetPluginHandler(Future<void> Function(String, ByteData?, PlatformMessageResponseCallback?) handler) {
+void webOnlySetPluginHandler(
+    Future<void> Function(String, ByteData?, PlatformMessageResponseCallback?)
+        handler) {
```

Comment:
Let's revert the auto-formatted changes. The engine has switched to the main Flutter style guide. I would also recommend turning off automatic formatting in the IDE:

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#:~:text=(The%20dart%20format%20tool%20uses%20the%20Dart%20guide%2C%20so%20we%20do%20not%20use%20it%20in%20the%20flutter/flutter%20and%20flutter/engine%20repositories.%20However%2C%20we%20do%20recommend%20its%20use%20in%20general.)

Reply:
done